src/hipscat/io/file_io/__init__.py

```diff
@@ -1,5 +1,6 @@
 from .file_io import (
     delete_file,
+    handle_pandas_storage_options,
```

Comment:
Let's not expose it in the `__init__`, since no one should be using the method on its own.